### ISSUE TITLE
EV customers should be scalable

### ISSUE THREAD

**Issue by jecollins** [powertac/powertac-server#788]
The EV model is a bottom-up model, and so has a scalability issue. If each customer represents a single vehicle, then it's hard to see how we could handle more than 100 or so instances. But each instance could represent multiple vehicles.

**Comment by jecollins** [comment]
Depends on #961, in order to provide useful statistical data.

**Comment by jecollins** [comment]
I suggest that the only truly scalable EV model would be a population model, driven by statistical models as in factored-customer.

**Comment by jecollins** [comment]
This is superseded by #1064
